src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -257,6 +257,7 @@ func CreateNode(
 	}
 	hostname := Name(nodeID)
 	cmd := []string{
+		"redpanda",
```

Comment:
? how is this possible :D 

does this change make sense? @0x5d

Reply:
Yeah. `rpk start` still works, but it shows a deprecation message. Thanks for doing this @BenPope , I had totally forgotten about it.